bc/assets/templates/includes/header.html

```diff
@@ -4,9 +4,12 @@
     <div class="max-w-7xl mx-auto px-4 sm:px-6 md:px-10">
         <div class="flex w-full justify-between items-center py-6 md:space-x-6 lg:space-x-10">
             <div class="absolute top-6">
-                <a href="{% url 'homepage' %}">
-                    <img alt='Yellow robot with white background logo' src="{% static "svg/icon.svg" %}"  layout="responsive" class="rounded-lg border-4 border-bcb-black h-20 w-[80px] lg:h-24 lg:w-24 bg-white"/>
-                </a>
+                    <div alt='Yellow robot with white background logo'  layout="responsive" class="rounded-lg border-2 p-2 border-gray-400 h-30 w-[120px] lg:h-32 lg:w-44 bg-white">
```

Comment:
This `alt` is no longer correct. Can you update it, please? 

Also, can you see if it's possible to avoid using the `px` classes and prefer using the other ones, with numbers, like `w-5` instead of `w-[40px]` or whatever? I'm not sure why, but I have a pretty strong feeling that using the px-based ones is bad form. Maybe somebody else knows? (You use these down below too.)

Reply:
The documentation comments that you can use the px values ​​to perfectly position the element: https://tailwindcss.com/docs/adding-custom-styles#using-arbitrary-values

I can use the other classes but it is not perfect, I used px adjustment so that when opening the responsive menu the logo is seen almost above the header logo